### ISSUE TITLE
Call to undefined method Laratrust\Laratrust::isAbleTo()

### ISSUE THREAD

**Issue by yorkcorpthe** [santigarcor/laratrust#687]
"php": "8.3.10",
    "jeroennoten/laravel-adminlte": "^3.14",
    "laravel-lang/common": "^6.4",
    "laravel/framework": "^11.31",    
    "laravel/ui": "^4.6",
    "santigarcor/laratrust": "^8.3"
},

**Describe the bug**
Laratrust example was just provided as reference on the documentation, maybe that method is now deprecated in that package, if you found which one should be used here, please share so we can update docs.
https://github.com/jeroennoten/Laravel-AdminLTE/wiki/Menu-Configuration

**To Reproduce**
![image](https://github.com/user-attachments/assets/e02934e9-bbd8-433d-ae57-ae63b5f8d57b)

1.Your custom filter needs to implement the FilterInterface provided by this package.

2.And then add the following configuration to the config/adminlte.php file:

![image](https://github.com/user-attachments/assets/fe93b4e7-258e-4d71-945a-7ccc273e9f26)
3.Register permission in menu 
![image](https://github.com/user-attachments/assets/ce994fdd-010f-4209-b4ee-69c5a6deeb11)
4.
![image](https://github.com/user-attachments/assets/240e867f-1941-4eac-9fb4-de5a8bc442c1)

**Comment by santigarcor** [comment]
Yeah, it's not defined in the Facade, you can use the `hasPermission` method or you're welcome to open a PR to add it to the Facade.
